Classes/Console/Command/Database/ExportCommand.php

```diff
@@ -14,29 +14,34 @@
  *
  */
 
-use Helhum\Typo3Console\Command\AbstractConvertedCommand;
 use Helhum\Typo3Console\Database\Configuration\ConnectionConfiguration;
 use Helhum\Typo3Console\Database\Process\MysqlCommand;
 use Helhum\Typo3Console\Database\Schema\TableMatcher;
-use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
-use Symfony\Component\Console\Output\ConsoleOutput;
 use Symfony\Component\Console\Output\OutputInterface;
 use TYPO3\CMS\Core\Database\ConnectionPool;
 use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Object\ObjectManager;
 
-class DatabaseExportCommand extends AbstractConvertedCommand
+class ExportCommand extends Command
 {
+    const OPT_EXCLUDE = 'exclude';
```

Comment:
What purpose do these constants serve? I'd rather avoid these or at least make them `private`.

Reply:
In general I get the idea. However, for now please use plain strings for option names. thanks

Reply:
I don't like having the same string two times or more used in the code, so I create a const. Currently witch PHP 7.0 I can not use private else I would have done ;) But if you don't like, I'll remove the consts...

Reply:
As I said, doing so is worth considering as it makes the option names a bit more explicit, but this should then be rolled out consistently over all commands. Since the latter is additional work, I'd prefer to stay consistent with this (little) duplication here.
So if you don't mind, I'd prefer plain strings for now. Thanks

Reply:
ofc will do. Just explained my intention to Mathias

Reply:
btw this should increase https://codeclimate.com/github/TYPO3-Console/TYPO3-Console/maintainability, isn't it?